packages/apps/src/microsoft/teams/apps/http_plugin.py

```diff
@@ -89,7 +93,16 @@ async def lifespan(_app: FastAPI) -> AsyncGenerator[None, None]:
             if self._on_stopped_callback:
                 await self._on_stopped_callback()
 
-        self.app = FastAPI(lifespan=lifespan)
+        @asynccontextmanager
+        async def combined_lifespan(app: Any):
```

Comment:
had to add this because other plugins may also want in on the lifespan in fastapi, and it this exposes it to other plugins and combines it when the app is run